_includes/translated-warning.html

```diff
@@ -1,5 +1,11 @@
+{% assign urlpathparts = page.url | split: "/" %}
+{% assign currlangcode = urlpathparts[1] %}
```

Comment:
Isn't it better to use `page.lang` here?

Reply:
Yes, indeed.